packages/connector-smtp/README.md

```diff
@@ -125,3 +159,141 @@ After going through the guide, your connector JSON should look like this:
 - [SendGrid - Integrating with the SMTP API](https://docs.sendgrid.com/for-developers/sending-email/integrating-with-the-smtp-api)
 - [Aliyun Direct Mail - Send emails using SMTP](https://www.alibabacloud.com/help/en/directmail/latest/send-emails-using-smtp)
 - [Aliyun Direct Mail - SMTP Reference](https://www.alibabacloud.com/help/en/directmail/latest/smtp-reference)
+
+# SMTP 连接器
+
+## 开始上手
+
+SMTP（简单邮件传输协议）是一种在网络中传输电子邮件的标准通信协议。邮件服务器和其他信息传输代理用 SMTP 来收发邮件。
+
+## 设置 SMTP 连接器
+
+SMTP 是一个所有邮件服务提供商通用的传输协议。
+
+我们提供了 SMTP 连接器用于以下几种邮件服务提供商的使用指南，帮助你有更好的理解：
+
+- _Gmail_ 是世界上最受欢迎的邮件服务商
+- _阿里云邮件_ 和 _SendGrid_（Logto 提供了对接这两个服务商的连接器，因此你们也许会对这两个服务商比较熟悉并有一个基本概念）
+
+希望在浏览了以下几个案例之后，你能够处理大部分其他邮件服务的配置 :rocket:
+
+### SMTP 连接器对接 Gmail 的配置
+
+你可以使用已有的 Gmail 邮箱或者通过 [Gmail](https://mail.google.com/) 创建一个新的账号。
+
+前往这篇 [Gmail 官方帮助文档](https://support.google.com/a/answer/176600)，查看如何得到 SMTP 连接器调用 Gmail 服务所 **必须的** 属性。
+
+根据文档的描述，并结合 Logto 的实现，你的 SMTP 连接器配置 JSON 应该如下：
+
+```json
+{
+    "host": "smtp.gmail.com",
+    "port": <port-number>,
+    "username": "<your-gmail-address>",
+    "password": "<password-to-previous-gmail-address>",
+    "fromEmail": "<your-gmail-address>",
+    "templates": [
+        {
+            "subject": "<register-template-subject>",
+            "content": "<Logto: Your passcode is {{code}}. (regitser template)>",
+            "usageType": "Register",
+            "contentType": "text/plain"
+        }
+    ]
+}
+```
+
+### 集成 SendGrid SMTP API
+
+我们假设你已经有了 SendGrid 账号，否则就在 [SendGrid](https://app.sendgrid.com/) 创建新账号。
+
+在 [集成 SendGrid SMTP API](https://docs.sendgrid.com/for-developers/sending-email/integrating-with-the-smtp-api) 可以找到详尽的集成指南。
+
+开发者可以前往 [_Sender Management_](https://mc.sendgrid.com/senders) 以访问相应的 _sender_ 详情。
+
+跟随配置指南，你的连接器配置 JSON 应该如下：
+
+```json
+{
+    "host": "smtp.sendgrid.net",
+    "port": 587,
+    "username": "apiKey",
+    "password": "<api-key-with-at-least-mail-permission>",
+    "fromEmail": "<email-address-of-a-verified-sender>",
+    "templates": [
+        {
+            "subject": "<register-template-subject>",
+            "content": "<Logto: Your passcode is {{code}}. (regitser template)>",
+            "usageType": "Register",
+            "contentType": "text/plain"
+        }
+    ]
+}
+```
+
+### 针对阿里云邮件帐号的配置
+
+登录 [阿里云](https://cn.aliyun.com/) 并创建新账号。
+
+跟随 [使用 SMTP 发送邮件](https://www.alibabacloud.com/help/zh/directmail/latest/send-emails-using-smtp)，完成里面的各个步骤以获取需要的配置和信息。
+
+前往 [SMTP 服务地址](https://www.alibabacloud.com/help/zh/directmail/latest/smtp-service-address) 并选择合适的 SMTP 服务地址与端口号。
+
+在 [阿里云工作台](https://dm.console.aliyun.com/) 侧边栏中找到「发信地址」，在这里你可以找到 `Sender address` 和 `SMTP Password`。
+
+跟随配置指南，你的连接器配置 JSON 应该如下：
+
+```json
+{
+    "host": "<SMTP-service-address>",
+    "port": <port-number>,
```

Comment:
it'll be better to use `jsonc` language and a mock number here to avoid json format failure:

```jsonc
{
  "port": 1234, // your SMTP port
}
```